packages/devtools_app/test/support/network_test_data.dart

```diff
@@ -465,3 +465,292 @@ final Map<String, dynamic> testSocket3Json = {
+  ...[32, 32, 32, 32, 123, 10, 32, 32, 32, 32, 32, 32, 116, 105, 116, 108, 101, 58, 32, 39, 102],
+  ...[111, 111, 39, 44, 10, 32, 32, 32, 32, 32, 32, 98, 111, 100, 121, 58, 32, 39, 98, 97, 114],
+  ...[39, 44, 10, 32, 32, 32, 32, 32, 32, 117, 115, 101, 114, 73, 100, 58, 32, 49, 44, 10, 32],
+  ...[32, 32, 32, 125, 10, 32, 32, 32, 32],
+];
+final httpPostResponseBodyData = [123, 10, 32, 32, 34, 105, 100, 34, 58, 32, 49, 48, 49, 10, 125];
+
+final httpPutRequest = HttpProfileRequest.parse(httpPutJson);
+final httpPut = DartIOHttpRequestData(0, httpPutRequest);
+final Map<String, dynamic> httpPutJson = {
+  'type': 'HttpProfileRequest',
+  'id': 1,
+  'isolateId': 'isolates/4447876918484683',
+  'method': 'PUT',
+  'uri': 'https://jsonplaceholder.typicode.com/posts/1',
+  'startTime': 1205283313,
+  'endTime': 1205859179,
+  'request': {
+    'events': [
+      {'timestamp': 1205855316, 'event': 'Connection established'}, 
+      {'timestamp': 1205858323, 'event': 'Request sent'}, 
+      {'timestamp': 1206602445, 'event': 'Waiting (TTFB)'}, 
+      {'timestamp': 1206609213, 'event': 'Content Download'},
+    ],
+    'headers': {
+      'transfer-encoding': [],
+    },
+    'connectionInfo': {
+      'localPort': 43684,
+      'remoteAddress': '2606:4700:3033::ac43:bdd9',
+      'remotePort': 443,
+    },
+    'contentLength': -1,
+    'cookies': [],
+    'followRedirects': true,
+    'maxRedirects': 5,
+    'method': 'PUT',
+    'persistentConnection': true,
+    'uri': 'https://jsonplaceholder.typicode.com/posts/1',
+    'filterKey': 'HTTP/client',
+  },
+  'response': {
+    'startTime': 1206603670,
+    'headers': {
+      'connection': ['keep-alive'],
+      'cache-control': ['no-cache'],
+      'date': ['Wed, 04 Aug 2021 08:57:24 GMT'],
+      'content-type': ['application/json; charset=utf-8'],
+      'pragma': ['no-cache'],
+      'access-control-allow-credentials': [true],
+      'content-length': [13],
+      'expires': [-1],
+    },
+    'compressionState': 'HttpClientResponseCompressionState.notCompressed',
+    'connectionInfo': {
+      'localPort': 43684,
+      'remoteAddress': '2606:4700:3033::ac43:bdd9',
+      'remotePort': 443,
+    },
+    'contentLength': 13,
+    'cookies': [],
+    'isRedirect': false,
+    'persistentConnection': true,
+    'reasonPhrase': 'OK',
+    'redirects': [],
+    'statusCode': 200,
+    'endTime': 1206609144,
+  },
+  'requestBody': httpPutRequestBodyData,
+  'responseBody': httpPutResponseBodyData,
+};
+final httpPutRequestBodyData = [
+  ...[32, 32, 32, 32, 123, 10, 32, 32, 32, 32, 32, 32, 116, 105, 116, 108, 101, 58, 32, 39, 102],
+  ...[111, 111, 39, 44, 10, 32, 32, 32, 32, 32, 32, 98, 111, 100, 121, 58, 32, 39, 98, 97, 114],
+  ...[39, 44, 10, 32, 32, 32, 32, 32, 32, 117, 115, 101, 114, 73, 100, 58, 32, 49, 44, 10, 32],
+  ...[32, 32, 32, 125, 10, 32, 32, 32, 32],
+];
+final httpPutResponseBodyData = [123, 10, 32, 32, 34, 105, 100, 34, 58, 32, 49, 48, 49, 10, 125];
+
```

Comment:
remove all but one new line here as well

Reply:
I'm so used to automatic formatting :-)